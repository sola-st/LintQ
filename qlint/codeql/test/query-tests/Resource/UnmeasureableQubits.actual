| op_added_with_for_loop.py:4:8:4:24 | ControlFlowNode for QuantumCircuit() | Circuit 'circ' has more qubits (12) than classical  bits (0) |
| op_added_with_for_loop.py:17:10:17:26 | ControlFlowNode for QuantumCircuit() | Circuit 'circ_x' has more qubits (6) than classical  bits (0) |
