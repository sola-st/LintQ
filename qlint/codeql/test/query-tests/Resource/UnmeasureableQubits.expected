| QL101_1_code_with_bug.py:5:11:5:36 | ControlFlowNode for QuantumCircuit() | Circuit 'circuit' has more qubits (3) than classical  bits (2) |
